client/src/main/java/io/split/engine/sse/dtos/GenericNotificationData.java

```diff
@@ -17,7 +22,10 @@ public GenericNotificationData (Long changeNumber,
                                     OccupancyMetrics occupancyMetrics,
                                     String segmentName,
                                     IncomingNotification.Type type,
-                                    String channel) {
+                                    String channel,
```

Comment:
the pcn, d and c naming is because we want to decrease the size payload in SplitChange notification. However, in the code you can use previousChangeNumber, featureFlagData and compressType. If you use what it comes, it makes difficult to developers without context understanding what is.

Reply:
Agree with this comment